deeplearning/src/main/java/nl/tudelft/bep/deeplearning/clustering/Cluster.java

```diff
@@ -89,7 +89,7 @@ public void calculateMean() {
 
 	/**
 	 * Looks at how many points (leafs of the tree) are in a cluster.
-	 * @return
+	 * @return the amount of points in this cluster
```

Comment:
Should be: 1 if the number of clusters is zero<br>
the number of clusters otherwise